### PR TITLE
django 1.4 patches for auth app

### PR DESCRIPTION
**Pull request by cckrause**
including broken tests and graceful degradation of older django versions

### CODE REVIEW

**Comment by cckrause** [comment]
- including fix for broken tests not broken tests ;)

File: django_couchdb_utils/auth/models.py [hunk 1]
```diff
@@ -2,13 +2,30 @@
...
 from django.core.mail import send_mail
 import random
 
 from . import app_label
 
+"""
+ django versions < 1.4 backward-compatibly patches
+"""
+
+if django_version[1] > 4:
```
Comment (stefankoegl) [inline django_couchdb_utils/auth/models.py]: On my local Django installation i get the following

```
>>> from django import VERSION as django_version
>>> django_version
(1, 4, 0, 'final', 0)
```

Therefore this wouldn't evaluate to True for me. Maybe something like

```
django_version[:2] > (1, 3)
```

would cover all cases?

Comment (cckrause) [inline django_couchdb_utils/auth/models.py]: yes thats right... my solution would struggle into problems in version code 2.x.. found another spelling bug....

**Comment by cckrause** [comment]
thats why i love open source :-)

**Pull request merged**